substrate/frame/support/src/lib.rs

```diff
@@ -869,7 +869,6 @@ macro_rules! hypothetically_ok {
 pub use serde::{Deserialize, Serialize};
 
 #[doc(hidden)]
-#[cfg(not(no_std))]
```

Comment:
This actually needed in both std and not std: https://github.com/paritytech/polkadot-sdk/pull/5150#discussion_r1692787358